packages/flutter/lib/src/cupertino/route.dart

```diff
@@ -1238,26 +1244,6 @@ Future<T?> showCupertinoModalPopup<T>({
 final Animatable<double> _dialogScaleTween = Tween<double>(begin: 1.3, end: 1.0)
   .chain(CurveTween(curve: Curves.linearToEaseOut));
 
-Widget _buildCupertinoDialogTransitions(BuildContext context, Animation<double> animation, Animation<double> secondaryAnimation, Widget child) {
```

Comment:
You moved this method to other place, that made it hard to see diff. Can you move it back to make code review easier?